server/Cargo.toml

```
@@ -1,2 +1,2 @@
 [workspace]
-members = ["svix-server", "svix-server_derive"]
+members = ["svix-server", "svix-server_derive", "svix_test_suite"]
```

```suggestion
members = ["svix-server", "svix-server_derive", "svix-test-suite"]
```
I know the _derive is inconsistent and confusing, but dashes are much prettier. :P